### DISCUSSION TITLE
Remote use - security?

### DISCUSSION THREAD

**Discussion by elandorr** [accetto/ubuntu-vnc-xfce-g3#59] [Q&A]
Hi there

if one wanted to run a process on a server (something bulky but still GUI dependent), is this setup useful? Or is it intended for local use?

VNC is unencrypted afaik. I found no reference to an SSH tunnel ad hoc.

I'm not aware of any "clean" solutions for linux like Teamviewer on windows that handle everything. (And TV is out of the question, being proprietary and already having been hacked before.)

Just curious :) Cheers

**Comment by mptagliari** [comment]
I'm on the same thing . A workaround is to use the image as a base into a docker compose and add the ssh variables with a script...

**Comment by markNZed** [comment]
You can configure it not to use VNC ports then access via noVNC which is a web based VNC client. That can be accessed over HTTPS so it is secure. In this way there is no need for SSH.
